webview-ui/test-generation/src/App.scss

```diff
@@ -68,6 +68,10 @@
   row-gap: 8px;
 }
 
+.app-not-loaded {
+  color: grey;
```

Comment:
Is there a `--vscode-*` color that's better for this? This grey doesn't stand out in a light theme.